app/pages/project/disks/DisksPage.tsx

```diff
@@ -162,6 +164,12 @@ export function DisksPage() {
     <>
       <PageHeader>
         <PageTitle icon={<Storage24Icon />}>Disks</PageTitle>
+        <DocsPopover
+          heading="Disks"
+          icon={<Storage16Icon />}
+          summary="Disks are persistent block-storage volumes that can be managed independently from VM instances."
```

Comment:
I think it would be "block storage" or "block-level storage" but not "block-storage". I'll make a note to fix that in the docs too. Regardless, in this case maybe we can drop it. It's sort an implementation detail.